libsolidity/formal/SSAVariable.cpp

```diff
@@ -27,16 +27,17 @@ using namespace dev;
 using namespace dev::solidity;
 
 SSAVariable::SSAVariable(
-	Declaration const& _decl,
+	Type const& _type,
+	string _uniqueName,
```

Comment:
Why not `const&` (also in many other places)?

Reply:
Forgot. Will fix.